scss/components/counter.scss

```diff
@@ -25,8 +26,9 @@ $block: #{$fd-namespace}-counter;
   text-align: center;
   padding: fd-space(1);
   &--notification {
+    font-size: $fd-counter-font-size;
```

Comment:
Why is there a font size declaration in here? This is just a modifier class used in conjunction with the base class.